api/v1alpha1/drill_types.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DrillSpec defines a benchmark run for drill load tester
+type DrillSpec struct {
+	// Image defines the drill docker image used for the benchmark
+	Image ImageSpec `json:"image"`
+
+	// BenchmarksVolume contains the files describing the benchmarks.
```

Comment:
BenchmarksVolume holds the content of benchmark files.
The key of the map specifies the filename and the value is the content of the file.
A ConfigMap is created from the map which is mounted as benchmarks
directory to the benchmark pod.